dlt/sources/rest_api/config_setup.py

```diff
@@ -711,7 +711,7 @@ def _expressions_to_resolved_params(expressions: Set[str]) -> List[ResolvedParam
         parts = expression.strip().split(".", maxsplit=2)
         if len(parts) != 3:
             raise ValueError(
-                f"Invalid definition of {expression}. Expected format:"
+                f"Invalid definition of `{expression}`. Expected format:"
                 " 'resources.<resource>.<field>'"
```

Comment:
nit: this file has a bunch of places where ' is still used. mostly in paths like these.